### PR TITLE
Stock management fixes

### PR DESCRIPTION
**Pull request by tulimaki**
Front: Show better error messages in basket update
Simple supplier: Make SKU-sort in stock management work
Simple supplier: Make admin filter+stock adjustment work
Simple supplier: Add djangojs.po file to transifex config

Refs SHOOP-1876

### CODE REVIEW

File: shoop/front/basket/update_methods.py [hunk 1]
```diff
@@ -14,6 +14,12 @@
 from shoop.utils.numbers import parse_decimal_string
 
 
+def get_error_text(error):
```
Comment (akx) [inline shoop/front/basket/update_methods.py]: What is this function supposed to do? It lacks a docstring.

File: shoop/front/basket/update_methods.py [hunk 2]
```diff
@@ -14,6 +14,12 @@
...
+    if hasattr(error, "__iter__"):
```
Comment (akx) [inline shoop/front/basket/update_methods.py]: I think this might be a little too broad. What are you trying to do here?

Comment (tulimaki) [inline shoop/front/basket/update_methods.py]: Trying to check if the error is iterable. We don't want to show errors like ["error 1", "error 2"] I understood that this isn't always the case so added the check. I will add docstring and tests for this function. Also I will rethink if this is even a good place for this function.

Comment (akx) [inline shoop/front/basket/update_methods.py]: Okay. I mean, the thing is even `strings` have `__iter__`, so if you passed in a string, you'd get `s,o,m,e,t,h,i,n,g,b,a,d` out. :D

**Pull request merged**